packages/design-system/blocks/navigation/data.js

```diff
@@ -1,7 +1,16 @@
 const data = [
   {
     title: "Fundacja",
-    href: "/fundacja",
+    links: [
+      {
+        title: "Fundacja",
```

Comment:
```suggestion
        title: "O nas",
```

The href is fine, but the title of this link should be "O nas"